pom.xml

```diff
@@ -24,6 +24,16 @@
     <developerConnection>scm:git:git@github.com:CruGlobal/idm-user-management.git</developerConnection>
     <tag>HEAD</tag>
   </scm>
+  <distributionManagement>
+    <repository>
+      <id>cruglobal.jfrog.io</id>
+      <url>https://cruglobal.jfrog.io/cruglobal/list/maven-cru-public-releases-local/</url>
+    </repository>
+    <snapshotRepository>
+      <id>cruglobal.jfrog.io</id>
+      <url>https://cruglobal.jfrog.io/cruglobal/list/maven-cru-public-snapshots-local/</url>
+    </snapshotRepository>
+  </distributionManagement>
```

Comment:
I would have used private instead of public. Is there anyone outside using idm-user-management?